app/Repositories/UserList/UserListRepository.php

```diff
@@ -41,4 +42,9 @@ public function model()
     {
         return UserList::class;
     }
+
+    public function attachPlace(UserList $userList, Place $place)
```

Comment:
Why did you use `find` if you have `getById` ? =)

Reply:
Fix that a little issue and resolve conflicts

Reply:
done)